### DISCUSSION TITLE
Testing events documentation should be updated to mention that it also applies to the new lifespan event

### DISCUSSION THREAD

**Discussion by mdw00d** [fastapi/fastapi#9238] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from contextlib import asynccontextmanager
import aiohttp
from fastapi import FastAPI, status, Response
from loguru import logger as log
from uvicorn import Config, Server

@asynccontextmanager
async def lifespan_session(app: FastAPI):
    async with aiohttp.ClientSession(raise_for_status=True) as session:
        log.info("Here I am")
        yield

app = FastAPI(lifespan=lifespan_session)

@app.get("/heartbeat", status_code=status.HTTP_204_NO_CONTENT, response_class=Response)
async def heartbeat() -> None:
    log.info("beat")

def run_server(host, port):
    server = Server(Config("authsvc.lifetest:app", host=host, port=port, log_level="info"))
    server.run()

if __name__ == "__main__":
    run_server("127.0.0.1", 8000)
```

### Description

In order to test using e.g. pytest FastAPI when the `lifespan` parameter is specified, it is necessary to follow the same procedures described in https://fastapi.tiangolo.com/advanced/testing-events/.

It would be helpful if that page was updated to also mention the new lifespan event so it will appear on searches; the page https://fastapi.tiangolo.com/ko/advanced/events/ could also be updated to point people to how test when using lifespan.

### Operating System

Windows

### FastAPI Version

0.93.0

### Python Version

3.10.7

**Comment by Kludex** [comment, marked as answer]
This is updated now.
